src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright 2017-2018 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "ngraph/runtime/intelgpu/intelgpu_backend.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+extern "C" void create_backend()
+{
+    runtime::Backend::register_backend("INTELGPU",
+                                       make_shared<runtime::intelgpu::IntelGPUBackend>());
+};
+
+shared_ptr<runtime::TensorView>
+    runtime::intelgpu::IntelGPUBackend::create_tensor(const element::Type& element_type,
+                                                      const Shape& shape)
+{
+    throw runtime_error("IntelGPUBackend::create_tensor: Not implemented yet");
```

Comment:
You can probably just steal the CPU implementation

Reply:
I will do this in separate PR with first implementation of the create_tensor() functions.